contracts/gateway/src/contract/migrations/v0_2_3.rs

```diff
@@ -0,0 +1,221 @@
+#![allow(deprecated)]
+
+use std::any::type_name;
+
+use axelar_wasm_std::error::ContractError;
+use axelar_wasm_std::nonempty;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Order, StdError, StdResult, Storage};
+use cw_storage_plus::{Key, KeyDeserialize, Map, PrimaryKey};
+use itertools::Itertools;
+use router_api::{Address, ChainName, ChainNameRaw};
+
+use crate::contract::{CONTRACT_NAME, CONTRACT_VERSION};
+use crate::state;
+
+const BASE_VERSION: &str = "0.2.3";
+
+pub fn migrate(storage: &mut dyn Storage) -> Result<(), ContractError> {
+    cw2::assert_contract_version(storage, CONTRACT_NAME, BASE_VERSION)?;
+
+    migrate_outgoing_messages(storage)?;
```

Comment:
you could also just drop the messages as well, they can always be reverifier and rerouted